### PR TITLE
Allow LATEST in get_limits

### LINKED ISSUE #679: Allow get_limits to use LATEST
This would allow for screens to do `LABELVALUELIMITSBAR INST LATEST TEMP1` which currently is unsupported

### PR DESCRIPTION
**Pull request by jmthomas**
closes #679

### CODE REVIEW

File: openc3/lib/openc3/api/limits_api.rb [hunk 1]
```diff
@@ -364,5 +364,35 @@ def _limits_group(group_name, action:, scope:, token:)
...
+    # @param scope [String] scope
+    # @return Hash The requested item based on the packet name
+    def _get_item(target_name, packet_name, item_name, scope:)
+      requested_item = nil
+      if packet_name == 'LATEST'
+        latest = -1
+        TargetModel.packets(target_name, scope: scope).each do |packet|
+          item = packet['items'].find { |item| item['name'] == item_name }
+          if item
+            # TODO: Fixme: This should be using the CVT not topics - Will possibly choose wrong packet if mixed with stored
```
Comment (ryanmelt) [inline openc3/lib/openc3/api/limits_api.rb:382]: Feels like a good opportunity to fix this?

**Review by ryanmelt — CHANGES_REQUESTED** [review]

File: openc3/lib/openc3/api/limits_api.rb (hunk 1 above)
Comment (jmthomas) [reply openc3/lib/openc3/api/limits_api.rb:382]: I didn't write that comment and I'm not sure what it means. The topic returns the time which we use to find 'latest'. The CvtModel only returns values.

Comment (ryanmelt) [reply openc3/lib/openc3/api/limits_api.rb:382]: The CVT stores the most recent of each packet.  The problem with topics is that they contain both realtime and stored data.  Whenever you use Topics you have to make sure the data you use isn't stored, and we aren't doing that here.

File: openc3/lib/openc3/api/limits_api.rb [hunk 2]
```diff
@@ -364,5 +364,34 @@ def _limits_group(group_name, action:, scope:, token:)
...
+    # @param scope [String] scope
+    # @return Hash The requested item based on the packet name
+    def _get_item(target_name, packet_name, item_name, scope:)
+      requested_item = nil
+      if packet_name == 'LATEST'
+        latest = -1
+        TargetModel.packets(target_name, scope: scope).each do |packet|
+          item = packet['items'].find { |item| item['name'] == item_name }
+          if item
+            hash = JSON.parse(Store.hget("#{scope}__tlm__#{target_name}", packet['packet_name']), :allow_nan => true, :create_additions => true)
```
Comment (ryanmelt) [inline openc3/lib/openc3/api/limits_api.rb:382]: This line should be a CvtModel method.

File: openc3/lib/openc3/api/limits_api.rb [hunk 3]
```diff
@@ -364,5 +364,34 @@ def _limits_group(group_name, action:, scope:, token:)
...
+    # @param target_name [String] target name
+    # @param packet_name [String] packet name
+    # @param item_name [String] item name
+    # @param scope [String] scope
+    # @return Hash The requested item based on the packet name
+    def _get_item(target_name, packet_name, item_name, scope:)
+      requested_item = nil
+      if packet_name == 'LATEST'
+        latest = -1
+        TargetModel.packets(target_name, scope: scope).each do |packet|
```
Comment (ryanmelt) [inline openc3/lib/openc3/api/limits_api.rb:379]: This whole loop should be a CvtModel or TargetModel method to get the latest item.

**Review by ryanmelt — CHANGES_REQUESTED** [review]

**Pull request merged**